dashboard/src/components/AppView/AppView.test.tsx

```diff
@@ -114,4 +122,36 @@ describe("AppViewComponent", () => {
       expect(sockets.length).toEqual(0);
     });
   });
+
+  describe("renderization", () => {
+    it("renders all the elements of an application", () => {
+      const wrapper = shallow(<AppViewComponent {...validProps} />);
+      expect(wrapper.find(ChartInfo).exists()).toBe(true);
+      expect(wrapper.find(DeploymentStatus).exists()).toBe(true);
+      expect(wrapper.find(AppControls).exists()).toBe(true);
+      expect(wrapper.find(ServiceTable).exists()).toBe(true);
+      expect(wrapper.find(AppNotes).exists()).toBe(true);
+      expect(wrapper.find(AppDetails).exists()).toBe(true);
+    });
+    it("renders a forbidden error if it exists", () => {
+      const wrapper = shallow(
+        <AppViewComponent {...validProps} deleteError={new ForbiddenError()} />,
+      );
+      expect(wrapper.find(PermissionsErrorAlert).exists()).toBe(true);
```

Comment:
we can put the result of `wrapper.find` in a variable so we don't try to find it twice